mobile-app/app/input.css

```diff
@@ -0,0 +1,51 @@
+@tailwind utilities;
+
+.font-light {
```

Comment:
@thedoublejay This one is little bit hack, BUT proper solution would require a lot of changes in our components, which I would recommend to do in next iteration

Reply:
Can you explain a bit what is the approach of this "hack"? And what would it take for future iterations

Reply:
Proper fix would be adding font family and weight in config
https://tailwindcss.com/docs/font-family
https://tailwindcss.com/docs/font-weight
And to use tailwind classes in our components.

Currently we have custom style classes that combine font family and weight into one class name.

So, this "hack" is adding fonts classes into root CSS file and being able to use them without changing components.

Reply:
Thanks for explaining, i'll create a placeholder ticket for now - https://github.com/DeFiCh/wallet/issues/3810